src/components/Main.js

```diff
@@ -30,9 +30,7 @@ export default class Main extends React.Component {
     // Create an observer to notify the changes in the scene
     var target = document.querySelector('a-scene');
     var observer = new MutationObserver(function (mutations) {
```

Comment:
I'm using `componentchanged` for components updates but when you delete a node for example from the dom inspector or from the console there's no way to catch that the DOM has been modified, so I check for mutations to refresh the scene, and right now I've a special case when the deleted entity (outside from the editor) is the current selected entity, so I need to update the selected entity to avoid inconsistent state